style.css

```diff
@@ -53,6 +63,40 @@ header {
   color: inherit;
 }
 
+.mobile-menu-close-icon {
+  color: #fff;
+  align-self: flex-end;
+  padding: 0;
+}
+
+.mobile-menu--hide {
+  display: none;
+  width: 0;
+  height: 0;
+}
+
+.mobile-menu--show {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding: 1em 0.75em;
+  width: 100vw;
+  height: 100vh;
+  color: #fff;
+  background: rgba(96, 112, 255, 0.9);
+  font-family: 'Poppins', sans-serif;
+  font-weight: 600;
+  font-size: 2rem;
+  line-height: 44px;
+  backdrop-filter: blur(4px);
+}
+
+.mobile-menu--show > ul > li {
+  padding: 0.5em 0;
+}
```

Comment:
- Great job implementing previous code reviewer suggestions,👍 I think you missed the desktop version of your navigation, when updating other things , please fix it for better UI 👌



| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/215725472-410f81e9-a3c0-4bcf-9c9b-1eef361fd658.png" width="600">|<img src="https://user-images.githubusercontent.com/52010789/215725466-adc1f2b9-2900-4c14-8ea4-b52a2c278713.png" width="600">|


😎️

🤝️